src/Covenant/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Covenant.Tests")]
```

Comment:
You can generate these with an `<ItemGroup>` in the `csproj` containing `<InternalsVisibleTo />` elements.

It's a stylistic choice, so if you prefer them in a file, this is fine, but I happen to prefer putting it in the `csproj`. To me it feels like the `csproj` is where we declare this component's relationship with other components. E.g., project and package references go in there. This is a relationship pointing in the other direction admittedly, but it still feels like something that belongs in there.